raven/processes/wps_raven.py

```diff
@@ -144,7 +147,15 @@ def _handler(self, request, response):
         kwds = self.options(request)
 
         # Launch model with input files
-        self.run(model, ts, kwds)
+        try:
+            self.run(model, ts, kwds)
+        except Exception as exc:
+            err_msg = traceback.format_exc()
+            # By default the error message is limited to 300 chars and strips
+            # many special characters
+            raise ProcessError(
```

Comment:
@huard @Zeitsperre Another possibility here would be to test for a `RavenError` here and only return its message, in this context.

Reply:
I think the traceback is useful, at least while we're still moving things around a lot.

Reply:
Sorry stupid question but what would happen before if `self.run(model, ts, kwds)` throws an exception? Where will it go?  Why do we have to catch and raise again?

Reply:
To add the traceback to the error message, and return the standard pywps ProcessError exception.